modules/ROOT/pages/manage-the-platform.adoc

```diff
@@ -40,9 +40,9 @@ After this step, the technical user will be able to connect to the engine and cr
 platformAPI.initializePlatform();
 ----
 +
-No users are created by default and the default username/password for the technical user to connect to Bonita Engine can be retrieved in the file `setup/platform_conf/current/tenants/[tenant_id]/tenant_engine/bonita-tenant-community-custom.properties` using the xref:bonita-bpm-platform-setup.adoc[plaform setup tool] (`[tenant_id]` being the default tenant ID, that is to say `1` when using the default configuration on a fresh installation).
+No users are created by default and the default username/password for the technical user to connect to Bonita Engine can be retrieved in the file `setup/platform_conf/current/tenants/[tenant_id]/tenant_engine/bonita-tenant-community-custom.properties` using the xref:bonita-platform-setup.adoc[plaform setup tool] (`[tenant_id]` being the default tenant ID, that is to say `1` when using the default configuration on a fresh installation).
```

Comment:
No users are created by default => No citizen users are created by default / No business users are created by default

Reply:
I added business, but Idk if this is really needed